concourse/templates/default.mako

```diff
@@ -231,7 +295,7 @@ else:
 % if job_step.retries():
   attempts: ${job_step.retries()}
 % endif
-  config:
+  config:${f' &{job_variant.variant_name}_{job_step.name}_config' if job_step._on_abort() else ''}
```

Comment:
stylistic nit: I think the following style would be easier to read:
```
% if job_step._on_abort():
  # ... 
% else:
  config:
% endif

also, I think it would really be nicer to refactor the step-coding